.github/workflows/release-client-api.yml

```diff
@@ -0,0 +1,65 @@
+name: Release api package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+          registry-url: https://npm.pkg.github.com/
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Install dependencies
+        run: |
+          cd tee-worker/client-api
+          corepack yarn install
+      - name: Build and Publish Parachain
+        run: |
+          branch_name="${{ github.ref }}"
+          echo "Branch Name: $branch_name" 
+          if [ "$branch_name" == "refs/heads/dev" ]; then
+            cd tee-worker/client-api/parachain-api
+            corepack yarn build
+            npm publish
```

Comment:
no `--tag dev` ?

Reply:
So, do we need three tags? `dev`, `staging`, and `latest`, right?

Reply:
Not sure we need a `latest` tag; I'd say one tag per repo mainline and that's it. Note that [this](https://github.com/litentry/litentry-parachain/issues/2130) will probably also become a mainline :thinking: